src/sentry/services/hybrid_cloud/integration/impl.py

```diff
@@ -210,6 +212,27 @@ def get_organization_integrations(
 
         return [serialize_organization_integration(oi) for oi in ois]
 
+    def organization_context(
+        self,
+        *,
+        organization_id: int,
+        integration_id: int | None = None,
+        provider: str | None = None,
+        external_id: str | None = None,
+    ) -> RpcOrganizationContext:
+        context = self.organization_contexts(
+            organization_id=organization_id,
+            integration_id=integration_id,
+            provider=provider,
+            external_id=external_id,
+        )
+        install = None
+        if context.organization_integrations:
+            install = context.organization_integrations[0]
```

Comment:
Do we need any validation here that we only retrieve a single organization integration?

Reply:
We could return the install only if the organizationid matches.